src/common/binaries_op/CMakeLists.txt

```diff
@@ -1,6 +1,11 @@
 add_library(binaries_op STATIC src/binaries_op.c)
 
+get_filename_component(COMMON_FOLDER ${CMAKE_CURRENT_SOURCE_DIR}/../ ABSOLUTE)
 target_include_directories(binaries_op PUBLIC include)
+include_directories(${COMMON_FOLDER}/pal/include)
+include_directories(${COMMON_FOLDER}/error_messages/include)
+include_directories(${COMMON_FOLDER}/utils/include)
+include_directories(${COMMON_FOLDER}/file_op/include)
```

Comment:
Done.
I have changed them all except for dbsync since its legacy code used them and I have not been able to make it to compile with target_include_directories.

Reply:
Done, except for dbsync